plugins/formal-verification/testData/diagnostics/no_contracts/inlining_captured.kt

```diff
@@ -0,0 +1,49 @@
+inline fun <!VIPER_TEXT!>invoke<!>(f: (Int) -> Int): Int {
```

Comment:
If you don't care about the generated code, you could use `NeverConvert` to drop this from the diagnostics output.